source/before-integrating/choose-which-user-attributes-your-service-can-request.html.md.erb

```diff
@@ -26,20 +26,17 @@ The user attributes and how you request them will depend on whether you are requ
     <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">You can only <a href=/before-integrating/choose-which-user-attributes-your-service-can-request/#choose-which-scopes-your-service-can-request><span>request user attributes using scopes</span></a>.</span></td>
   </tr>
   <tr>
-    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Authentication and <code>P1</code> identity assurance</span></td>
-    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">You can only <a href=/before-integrating/choose-which-user-attributes-your-service-can-request/#choose-which-scopes-your-service-can-request><span>request user attributes using scopes</span></a>.</span></td>
-  </tr>
-  <tr>
-    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Authentication and <code>P2</code> (or higher) identity assurance</span></td>
-    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">You can request user attributes using both claims and scopes (or either claims or scopes, depending on what your service needs).</span></td>
+    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Authentication and identity confidence</span></td>
```

Comment:
I'm a bit confused about this change - I thought authentication only could not use claims?

Reply:
Since we don't offer P1 at the moment, Phil suggested to take that line out, and replace the P2 line with Auth + Identity, since you can only request scopes with auth, but with auth + identity you can choose scopes and/or claims.

Reply:
Actually, we stuck with confidence here, because we are describing the concept, however, I am thinking we probably want to stay consistent and actually change it to proving after all, as we are talking about the type of request, and we refer elsewhere to auth and auth+identity, which is bringing us back to the core reason for having this PR. Would that make more sense?